egs/wsj/s5/steps/data/augment_data_dir.py

```diff
@@ -123,7 +124,7 @@ def CopyFileIfExists(utt_suffix, filename, input_dir, output_dir):
         if len(utt_suffix) > 0:
             new_dict = {}
             for key in dict.keys():
-                new_dict[key + "-" + utt_suffix] = dict[key]
+                new_dict[key + "_" + utt_suffix] = dict[key]
```

Comment:
I don't see why this needs to change.

Reply:
uh, I want to keep it consistent with reverberate_data_dir.py generated wav.scp.

Reply:
There are other scripts that use "-" for this purpose instead of "_", such as perturb_data_dir_speed.sh.

Anyway, we already have recipes that are using this script, so to prevent any potential back compatibility issues, it's better if you leave it how it was.

Reply:
OK, I will modify it to "-".